.gitignore

```diff
@@ -51,3 +51,6 @@ notes.txt
 
 # TypeScript
 tsout/
+
+# Generated icons
+ui/components/component-library/icons/lib
```

Comment:
The generated icon components shouldn't be committed to the repo but the svg files should be.